ods_ci/tests/Tests/400__ods_dashboard/413__ods_dashboard_user_mgmt.robot

```diff
@@ -49,28 +49,25 @@ Verify If Unauthorized User Can Not Change The Permission
 Verify Unauthorized User Is Not Able To Spawn Jupyter Notebook
     [Documentation]    Verify unauthorized User Is Not Able To Spawn Jupyter
     ...     Notebook , user should not see a spawner if the user is not in allowed Groups
+    ...     Note: this test creates users/groups via ODH Dashboard UI in
+    ...     `Settings -> User management` section. There is an another test that changes
+    ...     users/groups via `oc adm groups` command,see: `Verify User Can Set Custom RHODS Groups`
+    ...     in ods_ci/tests/Tests/500__jupyterhub/jupyterhub-user-access.robot
     [Tags]  ODS-1680
     ...     Tier1
     ...     Sanity
     Launch Dashboard And Check User Management Option Is Available For The User   ${TEST_USER.USERNAME}   ${TEST_USER.PASSWORD}  ${TEST_USER.AUTH_TYPE}
     Clear User Management Settings
-    Add OpenShift Groups To Data Science Administrators    rhods-users
-    Add OpenShift Groups To Data Science User Groups       rhods-users
+    Add OpenShift Groups To Data Science Administrators    rhods-admins
+    Add OpenShift Groups To Data Science User Groups       rhods-admins
```

Comment:
this won't likely solve the test problem, because by default RHODS has system:authenticated group as RHODS Users

Reply:
I think you're right - adding `Remove OpenShift Groups From Data Science User Groups    system:authenticated` before/after L62 should fix this

Reply:
The test passed just fine with this :thinking:

Reply:
@jstourac can you check the user mgmt settings in the Dashboard of your cluster?

Reply:
Okay, I know the reason - there is called `Clear User Management Settings` right above this. Didn't remember so I had to look into the code again.

Reply:
sounds good then. Thanks!